scrapers/bank_of_mauritius.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+
+import requests
+from playwright.sync_api import sync_playwright
+from utils import (
+    download_directory,
+    get_browser,
+    get_file_path,
+    get_md5hash,
+    get_standardized_filename,
+    parse_arguments,
+)
+
+BASE_URL = "https://www.bom.mu"
+CIRCULAR_PATH = "/financial-stability/supervision/guideline"
+
+
+def get_pdf_links(base_url: str, circular_path: str, count: int = 1) -> List[str]:
+    """Scraper function for circulars published by Bank of Mauritius
+
+    Args:
+        base_url (str): Home page of bank
+        circular_path (str): URL path containing circular publications
+        count (int, optional): Number of most recent circulars desired. Defaults to 1.
+
+    Returns:
+        List[str]: The URL path of the PDF
+    """
+    with sync_playwright() as pw:
+        browser = get_browser(pw)
+        page = browser.new_page()
+        page.goto(f"{base_url}{circular_path}")
+
+        # Since the PDFs are linked in each page individually,
+        # need to first page the page link
+        page_links = page.locator("div.col-sm-8 > a").all()[0:count]
+        links = [page_link.get_attribute("href") for page_link in page_links]
+
+        pdf_links = []
+        for link in links:
+            page.goto(f"{base_url}{link}")
+            pdf_icons = page.locator('img[src="/sites/default/files/pdf-icon.png"]').all()
```

Comment:
Unsure if this will work or not - what if you tried something along the lines of

```suggestion
            pdf_icons = page.locator('img[contains(@src, "pdf-icon.png")]').all()
```

Probably makes the locator a bit more robust against minute changes like how the folders are structured

Reply:
Unfortunately, this didn't work.  Would you happen to have a link to the Playwright docs for the `locator` method?  The [docs I've found](https://playwright.dev/python/docs/locators) are only on locator methods such as `get_by_role()`, etc..
Stack trace:
```
Exception has occurred: Error
Unsupported token "@src" while parsing selector "img[contains(@src, "pdf-icon.png")]"
  File "/Users/mlamchamkee/dev/scraper-infra/scrapers/bank_of_mauritius.py", line 43, in get_pdf_links
    pdf_icons = page.locator('img[contains(@src, "pdf-icon.png")]').all()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mlamchamkee/dev/scraper-infra/scrapers/bank_of_mauritius.py", line 62, in bank_of_mauritius_scraper
    link = get_pdf_links(BASE_URL, CIRCULAR_PATH)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mlamchamkee/dev/scraper-infra/scrapers/bank_of_mauritius.py", line 82, in <module>
    bank_of_mauritius_scraper()
playwright._impl._errors.Error: Unsupported token "@src" while parsing selector "img[contains(@src, "pdf-icon.png")]"
```

Reply:
Here's a link to the docs for the page.locator() method (between isClosed and mainFrame):
[https://playwright.dev/docs/api/class-page#page-locator](https://playwright.dev/docs/api/class-page#page-locator)

Reply:
Right - I forgot some syntax specifically for XPath.

Try some of the following:

```
# Best to have either single quotation on the outside and then double on the inside
# or vice versa. Not double quotation for both
page.locator('xpath=//img[contains(@src, "pdf-icon.png")]')
page.locator('//img[contains(@src, "pdf-icon.png")]')
```

Reply:
https://playwright.dev/docs/locators#locate-by-css-or-xpath